gc/base/SparseVirtualMemory.hpp

```diff
@@ -115,6 +121,7 @@ class MM_SparseVirtualMemory : public MM_VirtualMemory {
 	 */
 	bool freeSparseRegionAndUnmapFromHeapObject(MM_EnvironmentBase* env, void *dataPtr);
 
+	bool freeSparseRegionAndVerifyAndUnmapFromHeapObject(MM_EnvironmentBase *env, void *dataPtr, void *proxyObjPtr, uintptr_t size);
```

Comment:
like in DataPool class these 2 should be renamed so that Verify is after Update/Umap